pkgs/development/tools/packet-sd/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+buildGoModule rec {
+  pname = "prometheus-packet-sd";
+  version = "v0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "packethost";
+    repo = "prometheus-packet-sd";
+    rev = "v0.0.3";
+    sha256 = "sha256-2k8AsmyhQNNZCzpVt6JdgvI8IFb5pRi4ic6Yn2NqHMM=";
+  };
+
+  vendorSha256 = null;
```

Comment:
You need to set this unless there a no dependencies.

Reply:
There are no dependencies that have to be fetched. The build actually failed and told me to set this to null.